modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,285 @@
+master      rendered-master-e5e0c8e8be9194e7c5a882e047379cfa      True      False      False      3              3                   3                     0                      7d2h
+worker      rendered-worker-d6c9ca107fba6cd76cdcbfcedcafa0f2      True      False      False      3              3                   3                     0                      7d
+----
+=====
+
+. Create the following `SriovNetwork` object, and then save the YAML in the `sriov-network-attachment.yaml` file.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+ name: sriov-network
+ namespace: openshift-sriov-network-operator
+spec:
+ networkNamespace: test-namespace
+ resourceName: sriovnic
+ spoofChk: "off"
+ trust: "on"
+----
++
+[NOTE]
+=====
+See the "Configuring SR-IOV additional network" section for a detailed explanation on each option in `SriovNetwork`.
+=====
++
+An optional library, app-netutil, provides several API methods for gathering network information about a container's parent pod.
+
+. Create the `SriovNetwork` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-network-attachment.yaml
+----
+
+. Create a YAML file `tap-example.yaml` defining a network attachment definition, such as in the following example:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+ name: tap-one
+ namespace: test-namespace <1>
+spec:
+ config: '{
+   "cniVersion": "0.4.0",
+   "name": "tap",
+   "plugins": [
+     {
+        "type": "tap",
+        "multiQueue": true,
+        "selinuxcontext": "system_u:system_r:container_t:s0",
+        "owner": 1001,
+        "group": 1001
+     },
+     {
+       "type":"tuning",
+       "capabilities":{
+         "mac":true
+       }
+     }
+   ]
+ }'
+----
+<1> Specify the same `target_namespace` where the `SriovNetwork` object is created.
+
+. Apply the `yaml` by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f tap-example.yaml
+----
+
+. Create a performance profile by following the guidance in "Creating a performance profile".
+
+. Create the following `Pod` spec, and then save the YAML in the `dpdk-pod-rootless.yaml` file.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dpdk-app
+  namespace: test-namespace <1>
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+      {"name": "sriov-network", "namespace": "test-namespace"},
+      {"name": "tap-one", "interface": "ext0", "namespace": "test-namespace"}]'
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: "worker-0"
+  securityContext:
+      fsGroup: 1001 <2>
+      runAsGroup: 1001 <3>
+      seccompProfile:
+        type: RuntimeDefault
+  containers:
+  - name: testpmd
+    image: <DPDK_image> <4>
+    securityContext:
+      capabilities:
+        drop: ["ALL"] <5>
+        add: <6>
+          - IPC_LOCK
+          - NET_RAW #for mlx only <7>
+          - NET_ADMIN
```

Comment:
delete NET_ADMIN, @kononovn please confirm that it works without it. This should only be for manipulating routes with ip

Reply:
@kononovn Are you able to confirm this, I know you set to LGTM so may have to go with what is now there without confirmation.

Reply:
NET_ADMIN shouldn't be there. It goes against the purpose of rootless dpdk.

Reply:
removed